packages/workbox-core/_private.mjs

```diff
@@ -0,0 +1,5 @@
+import logger from './utils/logger.mjs';
+import WorkboxError from './models/WorkboxError.mjs';
+
+export {logger};
+export {WorkboxError};
```

Comment:
Out of curiosity, is this equivalent to

```js
export {logger, WorkboxError};
```

or is there a semantic difference when they're split into two `export` statements?